docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -111,6 +111,7 @@ target/generated-sources/grpc
         └── example
             ├── Greeter.java
             ├── GreeterBean.java
+            ├── GreeterClient.java
```

Comment:
Do we really have this? I thought we use a single interface both for client and server?

Reply:
Ah now I see it. That is the _implementation_. Cool.